docs/tasks/administer-cluster/configure-pod-disruption-budget.md

```diff
@@ -64,6 +64,8 @@ specified in the budget, thus bringing the number of available pods from the
 collection below the specified size. The budget can only protect against
 voluntary evictions, not all causes of unavailability.
 
+A `maxUnavailable` of 0% (or 0) or a `minAvailable` of 100% (or equal to the number of replicas) may block node drains entirely. This is permitted as per the semantics of `PodDisruptionBudget`. Cluster administrators may restrict this behavior through policy or admission control.
```

Comment:
I meant policy enforced by a particular organization. Hmm, I think that could be misinterpreted. I'll remove that.